packages/docs/command-line-interface/introduction.md

```diff
@@ -17,6 +17,7 @@ description: >-
 The Celo CLI is published as a node module on NPM. Assuming you have [npm installed](https://www.npmjs.com/get-npm), you can install the Celo CLI using the following command:
 
 ```bash
+apt-get install lsb-release libudev-dev libusb-dev libusb-1.0-0 -y
```

Comment:
are we sure we need these? I was able to do stuff on macOS without installing this -- might also confuse non-linux users

Reply:
Good question.. I just tested on a fresh mac and confirmed that these aren't necessary. I'll remove them here and add it to the Linux FAQ section below.